build.sh

```diff
@@ -13,7 +13,7 @@ trap cleanup INT TERM EXIT
 set -e
 
 echo ": Build and start containers..."
-export COMPOSE_FILE=docker-compose.test.yml
+export COMPOSE_FILE=docker-compose.yml
 docker-compose build
 
 # docker-compose on mac doesn't flush stdout and stderr properly without a tty,
```

Comment:
What do you think about the use just a `docker compose run` command with a shell script with a `depends_on` line in the `docker-compose.yml`, which encapsulates all of these lines below in a more readable format?

With this way there is no need to allocate tty.